ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -86,6 +90,7 @@ SettingsContentBase {
             id: main
 
             Layout.fillWidth: true
+            Layout.fillHeight: false
```

Comment:
@Khushboo-dev-cpp isn't `false` the default value for `Layout.fillHeight`? We don't need to set it explicitly (states for the same below).

Reply:
for StackLayout all children have Layout.fillWidth and Layout.fillHeight set to true https://doc.qt.io/qt-6/qml-qtquick-layouts-stacklayout.html#details:~:text=In%20contrast%20to%20most%20other%20layouts%2C%20child%20Items%27%20Layout.fillWidth%20and%20Layout.fillHeight%20properties%20default%20to%20true.%20As%20a%20consequence%2C%20child%20items%20are%20by%20default%20filled%20to%20match%20the%20size%20of%20the%20StackLayout%20as%20long%20as%20their%20Layout.maximumWidth%20or%20Layout.maximumHeight%20does%20not%20prevent%20it.